src/libs/ActiveClientManager/index.js

```diff
@@ -23,6 +39,16 @@ Onyx.connect({
  */
 function init() {
     Onyx.merge(ONYXKEYS.ACTIVE_CLIENTS, [clientID]);
+    didInitialize = true;
```

Comment:
Just to ensure there is no race condition between when `Onyx.connect` gets called and when `didInitialize` gets set to `true`, can you put this line above the `Onyx.merge`?